src/components/Navbar.jsx

```diff
@@ -94,37 +96,37 @@ function Navbar() {
   return (
     <nav
       ref={navRef}
-      className={`fixed left-0 top-0 z-10 w-full p-4 ${
-        activeLink === 'landing'
+      className={`fixed left-0 top-0 z-10 w-full p-4 ${activeLink === 'landing'
           ? 'bg-primary-400 text-sky-900'
           : 'bg-white shadow-lg'
-      }`}
-    >
-      <button
-        className="rounded border-2 px-4 lg:hidden"
-        onClick={() => setIsNavVisible(!isNavVisible)}
-      >
-        <FaBars className="h-10" />
-      </button>
-      <ul
-        className={`flex flex-col space-y-4 overflow-hidden lg:flex-row lg:justify-end lg:space-x-2 lg:space-y-0 lg:px-4 lg:py-2 ${
-          isNavVisible ? 'h-full' : 'h-0 lg:h-full'
         }`}
-      >
-        {sections.map((section) => (
-          <li key={section.id}>
-            <Link
-              to={`#${section.id}`}
-              onClick={(event) => handleNavigation(event, section.id)}
-              className={`${section.id === 'landing' ? 'hidden' : ''} p-6 ${
-                activeLink === section.id ? 'text-primary-500' : ''
-              }`}
-            >
-              {section.text}
-            </Link>
-          </li>
-        ))}
-      </ul>
+    >
+      <div className="flex items-center justify-between">
+        <img src={gdgDetroitLogo} alt="GDG Detroit Logo" className="h-16" />
```

Comment:
Image looks good - Not sure if we should add a white background to it in order to improve contrast of the image when the Navbar's background color is yellow